pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[build-system]
+requires = ["cffi", "setuptools", "wheel"]
```

Comment:
Should we also add `pycparser` to requires?

Reply:
I usually list the ones used explicitly here, and let `requirements/development.txt` list all of the ones they bring in.